tests/integration/tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist = flake8,rancher
 
 [testenv]
-basepython = python3.11
+basepython = python3.10
```

Comment:
Is this intentional?

Reply:
Yes, it's, GitHub runners comes with default python version 3.10, and it doesn't allow use to update the version to a newer one.